<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>geoXMCDA</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="styleSheet">
      <string notr="true">
font: 75 italic 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="CriteriaQTbox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>589</width>
        <height>538</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Table to XMCDA</string>
      </attribute>
      <widget class="QComboBox" name="CritListFieldsCBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>123</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="EnvLbl">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>111</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Active layer     </string>
       </property>
      </widget>
      <widget class="QLabel" name="CritFieldsLbl">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>122</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Active field</string>
       </property>
      </widget>
      <widget class="QLabel" name="CritMapNameLbl">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>361</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
         <weight>9</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QTableWidget" name="CritWeighTableWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>311</width>
         <height>251</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
      </widget>
      <widget class="QPushButton" name="CritRemoveFieldBtn">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>120</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QPushButton" name="CritExtractBtn">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>200</y>
         <width>101</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string>--&gt; XMCDA file</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SetHelpBtn">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>510</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
      <widget class="QPushButton" name="CritAddFieldBtn">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>120</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="toXMXCDAButtonBox">
       <property name="geometry">
        <rect>
         <x>365</x>
         <y>510</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel</set>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>461</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Output xmcda file</string>
       </property>
       <widget class="QToolButton" name="btnOutput">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>20</y>
          <width>61</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineOutput">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>351</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="IDFieldsLbl">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>460</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ID  field</string>
       </property>
      </widget>
      <widget class="QComboBox" name="IDListFieldsCBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>461</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CritListFieldsCBox</tabstop>
  <tabstop>CritAddFieldBtn</tabstop>
  <tabstop>CritRemoveFieldBtn</tabstop>
  <tabstop>CritWeighTableWidget</tabstop>
  <tabstop>CritExtractBtn</tabstop>
  <tabstop>SetHelpBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
